src/otel.ts

```diff
@@ -56,3 +59,10 @@ export const versions: readonly OtelVersion[] = [
 ];
 
 export const getLatestVersion = (): OtelVersion => versions[0];
+export const getVersion = (version: string | undefined): OtelVersion | undefined => {
```

Comment:
This is a major nit but should we rename this function to `getOtelVersion`? We use `import { versions as otelVersions}` exclusively and I wonder if we should rename that to `otelVersions` as well?